inference-engine/include/ie_blob.h

```diff
@@ -463,6 +515,22 @@ class INFERENCE_ENGINE_API_CLASS(MemoryBlob): public Blob {
  */
 using BlobMap = std::map<std::string, Blob::Ptr>;
 
+/**
+ * @brief This structure describes ROI data.
+ */
+struct ROI {
+    size_t id;     //!< ID of the ROI within the batch dimension (N)
+    size_t posX;   //!< W upper left coordinate of ROI
+    size_t posY;   //!< H upper left coordinate of ROI
+    size_t sizeX;  //!< W size of ROI
+    size_t sizeY;  //!< H size of ROI
```

Comment:
Is it possible to change members names to make their purpose clear without comments?

Reply:
I guess so, but it is certainly out of the scope. This is not refactoring PR - it solves particular task only

Reply:
It is not refactoring anyway since the structure is introduced in the PR, so I guess it is in scope..

Reply:
This is existing structure. It's moved upper only.

Reply:
Thanks for explanation